tools/SetupFlow/DevHome.SetupFlow/Views/PackageCatalogView.xaml.cs

```diff
@@ -55,6 +58,27 @@ public int GroupSize
     public PackageCatalogView()
     {
         this.InitializeComponent();
+    }
+
+    protected override AutomationPeer OnCreateAutomationPeer()
+    {
+        return new FrameworkElementAutomationPeer(this);
+    }
+
+    public void AddAllPackages()
+    {
+        var autoPeer = FrameworkElementAutomationPeer.FromElement(this);
+        if (autoPeer is not null)
+        {
+            var stringResource = new StringResource("DevHome.SetupFlow/Resources");
+            autoPeer.RaiseNotificationEvent(
+                AutomationNotificationKind.ActionCompleted,
+                AutomationNotificationProcessing.ImportantMostRecent,
+                stringResource.GetLocalized(StringResourceKey.AddAllAnnouncement, StringResourceKey.SelectedPackagesText),

```

Comment:
## Why is this PR in Draft?

Good question! I'm having trouble working with the `StringResource` stuff. I want the announcement to be localized, but the `stringResource` code I used seems to cause build issues.

I've tried inputting a string directly to line 77 (instead of getting the localized string) and that works great! So it looks liek this is the last bit (aside from any other feedback y'all give me).

Reply:
@carlos-zamora  I think this might be fixed in dev home v0.5 could you check if that build fixes this scenario. Thanks!